doc/news/_preparation_next_release.md

```diff
@@ -102,7 +102,14 @@ Known limitations:
 
 For more information see [Spec Plugin](../../src/plugins/spec/README.md).
 
-### <<HIGHLIGHT>>
+### Go binding into libelektra repository
+
+The [go binding](../../src/bindings/go-elektra/README.md) is now in the bindings folder.
+In your source code you can access the package with `github.com/ElektraInitiative/libelektra/go-elektra`.
+
+For more information and an example check the [README.md](../../src/bindings/go-elektra/README.md).
+
+> NOTE: On information how to publish new versions of `go-elektra` to go packages see https://go.dev/doc/modules/publishing.
 
```

Comment:
Please rework the rest of the document, too. (Remove duplicates, remove not used lines etc.)